ixofoundation/ixo-world
# chore: update top nav for ixo world (#72)

## Scope 
Update top nav for ixo.world homepage
https://nonaredteam.atlassian.net/secure/RapidBoard.jspa?rapidView=77&projectKey=IXO&modal=detail&selectedIssue=IXO-422

## Work done
Updated top nav for desktop and mobile view for ixo.world

## Steps to test
Verify that top nav matches designs
https://www.figma.com/file/ok50elV5abdYseQYKaSioG/ixo.world-website-revisions?node-id=3%3A0

Verify that links go to correct pages (note: all new pages don't exist yet)

– - Explorer (app.ixo.world) - Takes you to app.ixo.world
– - Build (developers.ixo.world) - Takes you to developers.ixo.world
– - Deliver (deliver.ixo.world) - Takes you to app.ixo.world/getixowallet/deliver
– - Invest (invest.ixo.world) - Takes you to app.ixo.world/getixowallet/invest
– -   Learn  (ixo-assistant FAQ) - Does not take you anywhere for now.

## Screenshot
- Desktop
<img width="615" alt="Screenshot 2020-05-18 at 11 34 52" src="https://user-images.githubusercontent.com/35291291/82197710-99a28680-98fb-11ea-82d5-f35d74b57ba9.png">

- Mobile
<img width="321" alt="Screenshot 2020-05-18 at 11 35 24" src="https://user-images.githubusercontent.com/35291291/82197769-acb55680-98fb-11ea-9d2b-3b3396583208.png">
